src/app/Fake.FluentMigrator/FluentMigrator.fs

```diff
@@ -0,0 +1,311 @@
+            context.Connection <- name
+            context.ConnectionStringConfigPath <- configPath
+
+let private setupConnectionForMode (context: IRunnerContext) mode =
+    match mode with 
+        | Execute(connection)
+        | ExecuteAndScript(connection, _) ->
+            setupConnection context connection
+        | Preview(connection) -> 
+            setupConnection context connection
+            context.PreviewOnly <- true
+        | Script(startVersion, _, _) ->
+            context.NoConnection <- true
+            context.StartVersion <- startVersion
+
+let private toRunnerContext task assemblies options = 
+    validate assemblies
+    let provider = getProviderFromTask task
+    let announcerFactory = 
+        match getModeFromTask task with
+            | Some(ExecuteAndScript(_, outputFileName)) ->
+                createAnnouncer outputFileName
+            | Some(Script(_, outputFileName, _)) ->
+                createAnnouncer outputFileName
+            | _ ->
+                createAnnouncer null
+    let announcer = announcerFactory options.Verbose provider
+    let context = new RunnerContext(fst announcer)
+    context.Database <- getProviderName provider
+    match task with
+        | MigrateUp(mode, None) ->
+            context.Task <- "migrate"
+            setupConnectionForMode context mode
+        | MigrateUp(mode, Some(version)) -> 
+            context.Task <- "migrate"
+            setupConnectionForMode context mode
+            context.Version <- version
+        | MigrateDown(mode, version) ->
+            context.Task <- "migrate:down"
+            setupConnectionForMode context mode
+            context.Version <- version
+        | Rollback(mode, steps) -> 
+            context.Task <- "rollback"
+            setupConnectionForMode context mode
+            context.Steps <- steps
+        | ListAppliedMigrations(connection) -> 
+            context.Task <- "ListAppliedMigrations"
+            setupConnection context connection
+    context.Targets <- Seq.toArray assemblies
+    context.Tags <- Seq.toList options.Tags
+    context.ApplicationContext <- options.Context
+    context.Profile <- options.Profile
+    context.ProviderSwitches <- options.ProviderSwitches
+    context.Timeout <- Nullable options.Timeout
+    context.TransactionPerSession <- options.TransactionPerSession
+    context.WorkingDirectory <- options.WorkingDirectory
+    match options.Namespace with
+        | Some(name, nested) -> 
+            context.Namespace <- name
+            context.NestedNamespaces <- nested
+        | None -> ()
+    (context, (snd announcer))
+
+/// Executes the specified task using configuration options
+/// ## Parameters
+///  - `task` - Database task to execute
+///  - `assemblies` - Assembly files which contain migrations
+///  - `options` - Migration options which are passed to FluentMigrator
+let ExecuteDatabaseTask task (assemblies: seq<string>) options =
```

Comment:
`executeDatabaseTask` (first letter is now small according to the fake 5 api guidelines, see https://fake.build/contributing.html#API-Design)